src/main/sensors/gyro.c

```diff
@@ -220,7 +222,19 @@ bool gyroInit(const gyroConfig_t *gyroConfigToUse)
         #endif
         }
     }
+
+    if (gyroConfig->gyro_soft_notch_hz) {
+        for (int axis = 0; axis < 3; axis++) {
+        #ifdef ASYNC_GYRO_PROCESSING
+            biquadFilterInitNotch(&gyroFilterNotch[axis], getGyroUpdateRate(), gyroSoftNotchHz, gyroConfig->gyro_soft_notch_hz);
```

Comment:
`gyroSoftNotchHz` is no longer defined.
I believe this call should be written in this way:
```
biquadFilterInitNotch(&gyroFilterNotch[axis], getGyroUpdateRate(), gyroConfig->gyro_soft_notch_hz, gyroConfig->gyro_soft_notch_cutoff_hz);
```